src/promptflow/promptflow/_core/run_tracker.py

```diff
@@ -216,7 +217,13 @@ def end_run(
     ):
         run_info = self._flow_runs.get(run_id) or self._node_runs.get(run_id)
         if run_info is None:
-            raise RunRecordNotFound(message=f"Run {run_id} not found", target=ErrorTarget.RUN_TRACKER)
+            raise RunRecordNotFound(
+                message_format=(
+                    f"Run record with ID {run_id} was not found in the storage. "
```

Comment:
Currently, the value of {run_id} is exposed to the message format. Is it expected?

Reply:
yes expected